common/compiler/solidity.go

```diff
@@ -187,15 +180,3 @@ func (s *Solidity) run(cmd *exec.Cmd, source string) (map[string]*Contract, erro
 	}
 	return contracts, nil
 }
-
```

Comment:
This is moved to common/compiler/helpers.go. :)

Reply:
@KimKyungup yes, geth have made helpers.go file and moved slurpFiles to the helpers.go file.